docs/reference/transform/setup.asciidoc

```diff
@@ -5,39 +5,95 @@
 <titleabbrev>Setup</titleabbrev>
 ++++
 
-To use the {transforms}, you must have the
-{subscriptions}[appropriate license] and at least one
-<<transform-setup-nodes,{transform} node>> in your {es} cluster. If {stack}
-{security-features} are enabled, you must also ensure your users have the
-<<transform-privileges,necessary privileges>>.
-
 [discrete]
-[[transform-setup-nodes]]
-== {transform-cap} nodes
+[[requirements-overview]]
+== Requirements overview
+
+To use {transforms}, you must have:
 
-To use {transforms}, there must be at least one {transform} node in your cluster.
-If you want to control which nodes run {transforms}, add or remove `transform`
-from the `node.roles` setting on some nodes. For more information, see
-<<modules-node>> and <<transform-settings>>.
+[%interactive]
+- [ ] the {subscriptions}[appropriate license]
+- [ ] at least one <<transform-node,{transform} node>>
+- [ ] security privileges assigned to users that:
++
+--
+* grant use of {transforms}, and
+* grant access to source and destination indices
+--
 
 [discrete]
 [[transform-privileges]]
 == Security privileges
 
-The {es} {security-features} provide <<built-in-roles,built-in roles>>
-and <<security-privileges,privileges>> that make it easier to control
-which users can manage or view {transforms}.
+Assigning security privileges affects how users access {transforms}. Consider 
+the two main categories:
 
-To _view_ the configuration and status of {transforms}, you must have:
+* *<<transform-es-security-privileges>>*: uses an {es} client, cURL, or {kib}
+**{dev-tools-app}** to access {transforms} via {es} APIs. This scenario requires
+{es} security privileges.
+* *<<transform-kib-security-privileges>>*: uses {transforms} in {kib}. This
+scenario requires {kib} feature privileges and {es} security privileges.
```

Comment:
It may make sense to emphasize the "and":
**and**

Reply:
Sure! I've updated in 63140b77aa69eb69981e9d32b7f26eb0c738b419